dev-db/mariadb-connector-odbc/mariadb-connector-odbc-3.1.11.ebuild

```
@@ -0,0 +1,65 @@
+# Copyright 2018-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+CMAKE_ECLASS=cmake
+
+inherit cmake-multilib flag-o-matic
+
+DESCRIPTION="MariaDB Connector/ODBC"
+HOMEPAGE="https://downloads.mariadb.org/connector-odbc/"
+SRC_URI="https://downloads.mariadb.org/interstitial/connector-odbc-${PV}/${P}-ga-src.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0/3.1"
+KEYWORDS="~amd64 ~x86"
+IUSE="gnutls ssl"
+
+S="${S}-ga-src"
+
+DEPEND="=dev-db/mariadb-connector-c-$(ver_cut 1-2)*
+	dev-db/unixODBC"
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	cmake_src_prepare
+
+	cp "${FILESDIR}/odbcinst.ini" . || die
+	sed -e "s,lib/lib,$(get_libdir)/lib,g" -i "odbcinst.ini" || die
```

You could combine this to a single `sed` call, 
```suggestion
sed -e "s,lib/lib,$(get_libdir)/lib,g" "${FILESDIR}/odbcinst.ini" > odbcinst.ini || die
```
but it doesn't matter. Just cosmetics.